Tutorials/GPU/CNS/Source/hydro/CNS_hydro_K.H

```diff
@@ -6,6 +6,8 @@
 #include <AMReX_FArrayBox.H>
 #include <cmath>
 
+using namespace amrex::literals;
```

Comment:
the only tiny comment: I would generally move those into function body scopes in header files. It's not too much harm here, but people will copy-paste from tutorials...

Reply:
Yes, will update it tonight.